plugin-testing-framework/definitions/replace.d.ts

```diff
@@ -0,0 +1,13 @@
+interface ReplaceOptions {
+	regex: string;
+	replacement: string;
+	paths: Array<string>;
+	recursive: boolean;
+	silent: boolean;
```

Comment:
ok then, but then is it ok to have like 3-4 copies of this file scattered around our repositories? should we put it on definitelytyped or something similar ourselves